frontend/src/host_orchestrator/orchestrator/createcvdaction.go

```diff
@@ -157,6 +158,12 @@ func (a *CreateCVDAction) launchWithCanonicalConfig(op apiv1.Operation) (*apiv1.
 			}
 		}()
 		args = append(args, "--credential_source="+filename)
+	} else if isRunningOnGCE() {
+		if ok, err := hasServiceAccountAccessToken(); err != nil {
+			return nil, fmt.Errorf("service account token check failed: %w", err)
```

Comment:
would it be better to just try to run `cvd` without credentials in this case?

Reply:
true, added a log message though.